docs/zkapps/tutorials/08-custom-tokens.mdx

```diff
@@ -0,0 +1,143 @@
+---
+title: "Tutorial 8: Custom Tokens"
+hide_title: true
+sidebar_label: "Tutorial 8: Custom Tokens"
+---
+
+:::info
+
+Please note that zkApp programmability is not yet available on Mina Mainnet, but
+zkApps can now be deployed to Berkeley Testnet.
+
+:::
+
+# Tutorial 8: Custom Tokens
+
+## Overview
+
+In this tutorial, we will discuss how to create custom tokens.
+
+Mina comes with native support for custom tokens. Each account on Mina can also have tokens associated with it.
+
+To create a new token, one creates a smart contract, which becomes the manager for the token, and uses that contract to set the rules around how the token can be mint, burned, and sent.
+
+The manager account may also set a token symbol for its token.
+
+In this tutorial, we will review smart contract code that creates and manages new tokens. You can find a full example, including these smart contracts and code to launch and interact with them, [here](https://github.com/es92/zkApp-examples/tree/main/08-custom-tokens/src).
```

Comment:
launch > deploy

Reply:
Is this the full example of what we're building in this tutorial? If so, might say:

> You can find the full code of what we'll be building in this tutorial, ...